### PR TITLE
Add Github actions for other architectures (arm64, s390x)

### PR DESCRIPTION
**Pull request by s-trinh**
See [Run-On-Arch GitHub Action](https://github.com/uraimo/run-on-arch-action)

Unfortunately, it takes lots of time to build but at least it should help detecting issues on exotic platforms.

### CODE REVIEW

File: .github/workflows/other-arch.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+    steps:
+    # https://github.com/marketplace/actions/cancel-workflow-action
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.5.0
+      with:
+        access_token: ${{ github.token }}
+
+    - name: Checkout repository
+      uses: actions/checkout@v2.1.0
```
Comment (cclauss) [inline .github/workflows/other-arch.yml:46]: Why an old release?  https://github.com/actions/checkout/releases
```suggestion
      uses: actions/checkout@v2
```

File: .github/workflows/other-arch.yml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+    - name: Run on arch
+      uses: uraimo/run-on-arch-action@v2.0.5
```
Comment (cclauss) [inline .github/workflows/other-arch.yml:49]: https://github.com/uraimo/run-on-arch-action/releases

File: .github/workflows/other-arch.yml [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+            distro: ubuntu20.04
+            target: POWER8
+          - arch: s390x
+            distro: ubuntu20.04
+            target: Z13
+
+    steps:
+    # https://github.com/marketplace/actions/cancel-workflow-action
+    - name: Cancel Previous Runs
+      uses: styfle/cancel-workflow-action@0.5.0
```
Comment (cclauss) [inline .github/workflows/other-arch.yml:41]: https://github.com/styfle/cancel-workflow-action/releases

**Comment by s-trinh** [comment]
@cclauss 
Thanks for the review!

See https://github.com/uraimo/run-on-arch-action/issues/64 for why we need to do `uraimo/run-on-arch-action@v2.1.1` currently.

**Comment by cclauss** [comment]
Please add uraimo/run-on-arch-action#64 in a comment to make things clear.

**Comment by s-trinh** [comment]
Let's merge this with only `aarch64` and `s390x` platforms and debug it later.

**Pull request merged**